validator/src/commands/run/args.rs

```diff
@@ -1099,12 +1099,14 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
     .arg(
         Arg::with_name("accounts_db_mark_obsolete_accounts")
             .long("accounts-db-mark-obsolete-accounts")
-            .help("Enables experimental obsolete account tracking")
+            .takes_value(true)
+            .possible_values(&["enabled", "disabled"])
             .long_help(
-                "Enables experimental obsolete account tracking. This feature tracks obsolete \
-                 accounts in the account storage entry allowing for earlier cleaning of obsolete \
-                 accounts in the storages and index. At this time this feature is not compatible \
-                 with booting from local snapshot state and must unpack from archives.",
+                "Controls the experimental obsolete account tracking. This feature tracks \
+                 obsolete accounts in the account storage entry allowing for earlier cleaning of \
+                 obsolete accounts in the storages and index. At this time this feature is not \
+                 compatible with booting from local snapshot state and must unpack from archives. \
```

Comment:
We also support fastboot now too. I'm fine if we update the text here in this PR.